adapters/appnexus/appnexus.go

```diff
@@ -280,6 +277,16 @@ func preprocess(imp *openrtb2.Imp, appnexusExt *openrtb_ext.ExtImpAppnexus, defa
 	return nil
 }
 
+func buildImpExt(appnexusExt *openrtb_ext.ExtImpAppnexus) *appnexusImpExt {
+	return &appnexusImpExt{Appnexus: appnexusImpExtAppnexus{
+		PlacementID:       int(appnexusExt.PlacementId),
+		TrafficSourceCode: appnexusExt.TrafficSourceCode,
+		Keywords:          makeKeywordStr(appnexusExt.Keywords),
+		UsePmtRule:        appnexusExt.UsePaymentRule,
+		PrivateSizes:      appnexusExt.PrivateSizes,
+	}}
+}
+
```

Comment:
Here buildImpExt is only initialises appnexusImpExt. This can happen in buildRequestImp. No need for separate function